### PR TITLE
Add environment-sensitive cross-site links to header.

### PR DESCRIPTION
**Pull request by leftclickben**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -158,6 +158,7 @@
                 .src('public/index.html')
                 .pipe(replace(/src="\/lib\/requirejs\/require\.js"/g, 'src="/application.js"'))
                 .pipe(replace(/href="\/css\/main\.css"/g, 'href="/main.css"'))
+                .pipe(replace(/<a href="(https?:\/\/)(.*)" class="cross-site-link">/g, '<a href="$1' + environment + '-$2" class="cross-site-link">'))
```
Comment (kehh) [inline gulpfile.js]: This looks brittle - when the environment changes to `production` the links will all be broken. Either leave a TODO here  or store the base url prefix with the environment (or whatever other solution you can think of).

Comment (kehh) [inline gulpfile.js]: Also in local development these urls will be completely wrong, but maybe that's ok.

Comment (leftclickben) [inline gulpfile.js]: It's meant to skip these steps in production so the URLs in the source file remain as is, but obviously I forgot the skipping logic, will fix. The `package` tasks don't run in development.

Comment (leftclickben) [inline gulpfile.js]: (...which means the links in dev point to production, which is fine)

Comment (kehh) [inline gulpfile.js]: :+1:

**Pull request merged**